Xamarin.Forms.Platform.iOS/Cells/EntryCellRenderer.cs

```diff
@@ -23,6 +23,11 @@ public override UITableViewCell GetCell(Cell item, UITableViewCell reusableCell,
 				tvc.KeyboardDoneButtonPressed -= OnKeyBoardDoneButtonPressed;
 			}
 
+			tvc.SelectedBackgroundView = new UIView()
```

Comment:
Do not set this unless item.SelectedBackgroundColor is set to something OTHER than Color.Default. If it gets set back to Color.Default it needs to be restored to its proper color. Also I don't see you doing change tracking anywhere which is a problem.

Reply:
@jassmith I was trying to work on tracking the change and updating the value but It is not hitting the HandlePropertyChanged. Can you suggest me why?

Reply:
and also already existing code for monitoring the change in imagesource in ImageCell is also not working.

Reply:
we should fix that then...

Get the Default issue fixed first.